architecture/2025-04-compute-extensibility.md

```diff
@@ -0,0 +1,469 @@
+# Compute Platform Extensibility
+
+* **Author**: Brooke Hamilton (@Brooke-Hamilton)
+
+## Overview
+
+Radius will enable users to deploy and manage their applications to multiple compute platforms, in addition to Kubernetes, by adding support for recipes to the core resource types. Recipes will enable the Radius team to provide default implementations for multiple platforms, and allow customers to modify or provide their own deployment recipes.
+
+Modifications to Radius include:
+
+* The [core resource types](https://docs.radapp.io/reference/resource-schema/core-schema/) of application, container, gateway, and secret store will support recipes.
```

Comment:
Why applications? I see the use cases for the others

Reply:
I will remove recipes from applications.